api/index.ts

```diff
@@ -15,52 +21,86 @@ import apqWeather from '@trygql/apq-weather';
 import uploadsMock from '@trygql/uploads-mock';
 import webCollections from '@trygql/web-collections';
 
-const indexPath = path.join(process.cwd(), 'page/dist/index.html');
-const indexFile = fs.promises.readFile(indexPath);
+const bootApp = async () => {
+  const indexPath = path.join(process.cwd(), 'page/dist/index.html');
+  const indexFile = await fs.promises.readFile(indexPath);
 
-const app = Fastify({
-  logger: {
-    level: 'warn',
-  }
-});
-
-app.register(headers);
-app.register(context);
-app.register(underPressure, { maxEventLoopDelay: 1000 });
+  const app = Fastify({
+    logger: {
+      level: 'warn',
+    },
+    pluginTimeout: 2 * 60 * 1000, // 2 minutes
+  });
 
-app.register(basicPokedex);
-app.register(intermittentColors);
-app.register(relayNpm);
-app.register(apqWeather);
-app.register(uploadsMock);
-app.register(webCollections);
+  app.register(headers);
+  app.register(context);
+  app.register(underPressure, {
+    maxEventLoopDelay: 1000,
+    pressureHandler: (req, rep, type, value) => {
+      switch (type) {
+        case TYPE_HEAP_USED_BYTES:
+          app.log.warn(`too many heap bytes used: ${value}`);
+          break;
```

Comment:
Add some logging when we hit certain boundaries. I'm wondering if we're running out of memory on our instances, hoping this will help test that hypothesis.